### ISSUE TITLE
损失特别大有啥优化策略吗

### ISSUE THREAD

**Issue by 2017TJM** [bing0037/keras-yolo3#4]
Epoch 38/50
4/4 [==============================] - 10s 2s/step - loss: 92487.3184 - val_loss: 75600.4219
Epoch 39/50
4/4 [==============================] - 10s 3s/step - loss: 89534.9199 - val_loss: 63017.3125
Epoch 40/50

**Comment by AGrosserHH** [comment]
How can this be improved?
